tkweb/apps/drinks/views.py

```diff
@@ -0,0 +1,31 @@
+from django.http import HttpResponseRedirect
+from django.shortcuts import render, get_object_or_404
+from django.views.generic.edit import FormView
+
+from .forms import BarcardGenForm
+from .models import Barcard
+
+class BarcardSelect(FormView):
+    template_name = 'drinks/home.html'
+    form_class = BarcardGenForm
+    success_url = 'generate/'
+
+    def form_valid(self, form):
+        return super().form_valid(form)
+
+def barcardGen(request):
+    if request.method =='POST':
+        card = request.POST.get('barcard')
+        card_obj = Barcard.objects.get(id=card)
```

Comment:
Her bør vi give en 404 hvis man giver et forkert/ugyldigt ID:
```suggestion
        card_obj = get_object_or_404(Barcard, id=card)
```